src/modules/simulator/simulator.h

```diff
@@ -353,15 +353,16 @@ class Simulator : public ModuleParams
 	// class methods
 	int publish_sensor_topics(mavlink_hil_sensor_t *imu);
 	int publish_flow_topic(mavlink_hil_optical_flow_t *flow);
-	int publish_ev_topic(mavlink_vision_position_estimate_t *ev_mavlink);
+	template<typename T>
```

Comment:
Why use a template?

Reply:
both msgs endup having the same uORB msg structure to fill - so, to avoid repeating code, a template works much better.